src/olympia/reviews/views.py

```diff
@@ -166,7 +166,11 @@ def delete(request, addon, review_id):
     review = get_object_or_404(Review.objects, pk=review_id, addon=addon)
     if not user_can_delete_review(request, review):
         raise PermissionDenied
-    review.delete(user_responsible=request.user)
+    # Log deleting moderated reviews
+    if review.editorreview:
+        review.moderator_delete(user=request.user)
```

Comment:
Ah, I didn't see a different method (`moderator_delete`) was called.  Idea: change `Review.delete` to do a `review.editorreview` check?  But you'll need to check the user deleting isn't the review creator.